dashboard/app/controllers/api/v1/pd/workshop_survey_foorm_submissions_controller.rb

```diff
@@ -11,13 +11,16 @@ def create
     pd_session_id = params[:pd_session_id].blank? ? nil : params[:pd_session_id].to_i
     day = params[:day].blank? ? nil : params[:day].to_i
     form_name = params[:form_name].presence
+    # agenda will be blank if none was provided, we would like to store it as null
```

Comment:
Does "blank" mean an empty string, like ""?

Reply:
yep!

Reply:
I'll make the comment more clear